src/includes/getting-started-install/native.qt.mdx

```diff
@@ -0,0 +1,50 @@
+The Qt integration is part of the [`sentry-native`](https://github.com/getsentry/sentry-native/) SDK,
+which currently supports Windows, macOS, Linux, and Android. Both Qt 5 and Qt 6 is supported.
+
+To build the SDK, download the latest release of the SDK from the [Releases page](https://github.com/getsentry/sentry-native/releases). The SDK is managed as a [CMake] project, which additionally supports several configuration options, such as the backend to use.
```

Comment:
missing link for CMake?
```suggestion
To build the SDK, download the latest release of the native SDK from the [Releases page](https://github.com/getsentry/sentry-native/releases). The SDK is managed as a [CMake] project, which additionally supports several configuration options, such as the backend.
```

Reply:
This is copied directly from the original native SDK docs. But looking at it now I think we can simplify it (in both places) to just "To build the SDK, download the latest release from the [Releases page]". What do you think? We're already talking about the native SDK, so no need to repeat it.

Reply:
Or perhaps "download the latest sources from the [Releases page]", since we're talking about sources here, not a binary release?

Reply:
Or is https://github.com/getsentry/sentry-native/releases/download/0.4.8/sentry-native.zip different from the source archive? Does the former include submodules?

Reply:
> missing link for CMake?

I think this is already handled by the link defined below? Works when rendered at least

Reply:
Yes, I think "download latest sources..." is both more accurate and simpler

for CMake I didn't see the link, just the square brackets surrounding it on that line, thus my q

Reply:
Cool, will update. See comment below for cmake link